homeassistant/util/unit_conversion.py

```diff
@@ -358,16 +369,24 @@ class VolumeConverter(BaseUnitConverter):
     _UNIT_CONVERSION: dict[str, float] = {
         VOLUME_LITERS: 1 / _L_TO_CUBIC_METER,
         VOLUME_MILLILITERS: 1 / _ML_TO_CUBIC_METER,
-        VOLUME_GALLONS: 1 / _GALLON_TO_CUBIC_METER,
-        VOLUME_FLUID_OUNCE: 1 / _FLUID_OUNCE_TO_CUBIC_METER,
+        VOLUME_UK_GALLONS: 1 / _UK_GALLON_TO_CUBIC_METER,
+        VOLUME_UK_FLUID_OUNCE: 1 / _UK_FLUID_OUNCE_TO_CUBIC_METER,
+        VOLUME_US_GALLONS: 1 / _US_GALLON_TO_CUBIC_METER,
+        VOLUME_US_FLUID_OUNCE: 1 / _US_FLUID_OUNCE_TO_CUBIC_METER,
+        VOLUME_GALLONS: 1 / _US_GALLON_TO_CUBIC_METER,
+        VOLUME_FLUID_OUNCE: 1 / _US_FLUID_OUNCE_TO_CUBIC_METER,
```

Comment:
Should these two be removed, and fail conversion?
Should they be included, but raise a warning in the logs?
Or should they still be included without a warning?